extension/extension.cpp

```diff
@@ -45,7 +45,7 @@ typedef enum EAuthProtocol
 	k_EAuthProtocolSteam = 3,
 } EAuthProtocol;
 
-#if SOURCE_ENGINE < SE_SDK2013 || SOURCE_ENGINE == SE_TF2 || SOURCE_ENGINE == SE_LEFT4DEAD || SOURCE_ENGINE == SE_LEFT4DEAD2
+#if SOURCE_ENGINE == SE_LEFT4DEAD || SOURCE_ENGINE == SE_LEFT4DEAD2

```

Comment:
Sorry I'm not updated on what's available in the new sdk. But are you sure the steam headers are missing, we need to define the enums ourselves ?

Reply:
EBeginAuthSessionResult is already defined for newest sdks so not needed

Reply:
Right you're correct I read that change wrong. This is indeed fine to remove.